### DISCUSSION TITLE
gh-pages branch never deployed

### DISCUSSION THREAD

**Discussion by jcreek** [community/community#36836] [Other Feature Feedback, Questions, & Ideas] [answered]
I've set up an action as specified in the mkdocs documentation at https://github.com/jcreek/jcreek.github.io and the gh-pages branch never gets picked up for deployment - any ideas as to why?

**Comment by jcreek** [comment, marked as answer]
I've solved my own issue - I needed to manually specify to build based on changes on the gh-pages branch.

**Comment by ossanna16** [comment]
Thank you for asking a good question @jcreek, I'm glad you found our community 🙂
